test/robot/web/www/js/actions.js

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
If you haven't already, please can you run these javascript files through JSLint? There's a [VSCode plugin](https://github.com/ajhyndman/vscode-jslint).

Reply:
jslint doesn't like classes